Configuration/DataProcessing/python/Impl/ppEra_Run2_2017_ppRef.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+_ppEra_Run2_2017_ppRef_
+
+Scenario supporting proton collisions
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Reco import Reco
+import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Run2_2017_ppRef_cff import Run2_2017_ppRef
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class ppEra_Run2_2017_ppRef(pp):
+    def __init__(self):
+        pp.__init__(self)
+        self.recoSeq=''
+        self.cbSc='pp'
+        self.addEI=True
+        self.isRepacked=True
```

Comment:
oops. This is wrong.
It should be set to false.
https://hypernews.cern.ch/HyperNews/CMS/get/tier0-Ops/1776/1/1/1/1/1/1/1/2/1/1/2/1/1/1.html

Reply:
Do you want a new 92X PR for this ? Also should there be new PRs for 94X and master branch ?

Reply:
I made a PR for 92X and for master.
There is no need for 94X because T0 is not going to switch to 94X and this scenario.

Reply:
PRs with fixes are in #21255 and #21256